packages/@lightningjs/ui-components/src/components/TextBox/TextBox.stories.js

```diff
@@ -71,6 +75,22 @@ Basic.argTypes = {
     table: {
       defaultValue: { summary: false }
     }
+  },
+  fixed: {
+    control: 'boolean',
+    description:
+      'Flag that when set to `true`, allows the width of the component to be set with `w`',
+    table: {
+      defaultValue: { summary: 'undefined' }
+    }
+  },
+  w: {
+    control: 'number',
+    description:
+      'When the `fixed` property is `true`, this will set the width of the component',
```

Comment:
Should we specify here that, it will only set the width to the `w` value (assuming `fixed` is `true`) if `wordWrapWidth` is not defined?

**If `wordWrapWidth` is defined at all, the width will reflect the `wordWrapWidth` even if a `w` value is provided and `fixed` is set to `true`

Reply:
agreed